tests/integration/test_magicc7.py

```diff
@@ -294,3 +295,28 @@ def test_return_config(test_scenarios, magicc7_is_available, out_config):
             assert set(ssp126.get_unique_meta(k)) == set(rf_total_runmoduses)
         else:
             raise NotImplementedError(k)
+
+
+def test_return_config_clash_warning(test_scenarios, magicc7_is_available, caplog):
```

Comment:
Will this still work with openscm/magicc#321 now raising a `ValueError` on duplicate keys?

Reply:
Nope, I wasn't going to release pymagicc for a bit

Reply:
So this could go in, then when we update pymagicc, we update this too and pin pymagicc I guess